app/views/projects/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Project Show Page</h1>
+
+<p><%= @project.name%></p>
+<p>Material: <%= @project.material%></p>
+<p>Challenge Theme: <%= @theme%></p>
+<p>Number of Contestants: <%= @count%></p>
+<p>Average Contestant Experience: <%= @exp%></p>
+
+<p> Add Contestant To Project</p>
+<%= form_with url: "/projects/#{@project.id}", method: :patch, local: true, data: {turbo: false} do |f| %>
```

Comment:
Here is where we would want our form to create a new relationship between and existing project and contestant vs updating (:patch) a project. So it would look like:
```ruby
<%= form_with url: "/project/#{@project.id}/contestants", method: :post, local: true do |form| %>
 <%= form.label :contestant_id %>
 <%= form.number_field :contestant_id %>

 <%= form.submit "Add Contestant To Project" %>
<% end %>
```
A good way to know when to do this is when you read this:
```
I see a form to add a contestant to this project
When I fill out a field with an existing contestants id
```
you are adding an existing contestant to an existing project.